scripts/create_synthetic_data.py

```diff
@@ -0,0 +1,53 @@
+""" Script to create synthetic data on the basis of backtranslation
+with Google translate. Use as follows:
+
+> python scripts/create_synthetic_data.py --lang nl --filename data_file.txt
+
+lang: the language via which to do the backtranslation
+data_file: the source file that will be translated line per line
+
+To connect with Google Translate, the environment variable
+GOOGLE_APPLICATION_CREDENTIALS needs to refer to the key file
+"""
+
+import os
+
+import ndjson
+import click
+from google.cloud import translate
+
+# Instantiates a client
+translate_client = translate.Client()
+
+
+@click.command()
+@click.option('--lang', help='The language via which to do the backtranslation')
+@click.option('--filename', help='The input file')
+def generate(filename, lang):
```

Comment:
For ease of use, we might eventually want to do all of the synthetic translations at once (for say, 10 languages) and dump them into a single file, maybe with a timestamp in its name.